package.json

```diff
@@ -9,6 +9,8 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@clerk/backend": "^0.37.0",
```

Comment:
why did you add these 2?

Reply:
i think i added this back when i was trying to get the enviorment to run and the suggestion i got from searching online was to make sure i had the latest clerk installed. I can remove